AsyncDisplayKit/ASTableView.mm

```diff
@@ -737,6 +737,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
   ASCellNode *node = [_dataController nodeAtIndexPath:indexPath];
+  [node layoutIfNeeded];
```

Comment:
I think I commented on the collection view one for this, but we should think about if it is possible to  eliminate this and allow the wrapping cell to do it's normal layout process and setting of the frame.